multi-node-k8s.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
```

Comment:
This version (currently 1.18.3) might have an incompatibility with snapshots? @Madhu-1 can you confirm and let us know what version you recommend for the current e2e tests?